src/components/collapsible/Collapsible.js

```diff
@@ -119,9 +119,11 @@ class Collapsible extends React.Component<Props, State> {
                     </PlainButton>
                     {isOpen && headerActionItems}
                 </div>
-                <AnimateHeight duration={animationDuration} height={isOpen ? 'auto' : 0}>
-                    <div className="collapsible-card-content">{children}</div>
-                </AnimateHeight>
+                {isOpen && (
+                    <AnimateHeight duration={animationDuration} height="auto">
```

Comment:
Does animation still work properly with this change? React can sometimes struggle with animating a "new" item in the DOM.

Reply:
Yeah it seems to work fine.

Reply:
I'm seeing the cards fail to animate if I make the same change locally, likely due to the class being applied after the element has been added to the DOM at full height. Is it possible the react-animate-height library supports this already in a different way?